### PR TITLE
feat: add instagram data source to oracle script

### PR DESCRIPTION
**Pull request by dadamu**
This PR adds Instagram data source to oracle script.

## Checklist
- [x] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [ ] Updated the documentation. 
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: oracle-scripts/src/script.rs
Comment (dadamu) [inline oracle-scripts/src/script.rs:11]: We didn't upload the Instagram data source to main net since I don't have mainnet deployer key.

Comment (RiccardoM) [reply oracle-scripts/src/script.rs:11]: I think we can wait until the data source has been deployed to merge this.

Comment (dadamu) [reply oracle-scripts/src/script.rs]: Updated

**Review by RiccardoM — APPROVED** [review]

**Pull request merged**
